packages/apps-v5/src/commands/ps/index.js

```diff
@@ -31,7 +31,7 @@ function printExtended (dynos) {
 }
 
 async function printAccountQuota(context, heroku, app, account) {
-  if (app.process_tier !== 'free') {
+  if (app.process_tier !== 'eco') {
```

Comment:
Apps can still be on the free tier until the end of the month if the customer isn't subscribed to Eco. We'll need this to support both the free and eco cases while free is still in the system.

Reply:
👍